clouddq/classes/dq_rule_binding.py

```diff
@@ -334,11 +336,20 @@ def resolve_all_configs_to_dict(
             row_filter_config = self.resolve_row_filter_config(configs_cache)
             # resolve reference columns config
             if self.reference_columns_id:
+                include_all_reference_columns = False
                 include_reference_columns = self.resolve_reference_columns_config(
                     configs_cache
                 ).include_reference_columns
+                if include_reference_columns[0] == "*":
```

Comment:
Can we also check if the length of the include_reference_columns is 1?

Reply:
in case we have both * and few other columns in reference columns section, I should raise an error or log a warning and continue with execution with including reference columns.

Reply:
I think it's probably better to raise an error if the list include "*" and other columns.

Reply:
agree we should check and throw error if user include "*" and other columns